src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -286,6 +307,28 @@ bool AudioUnitEffect::InitializePlugin()
    return true;
 }
 
+class AudioUnitValidator : public DefaultEffectUIValidator {
```

Comment:
Not sure if I remember this correctly, anyway: we said we should use EffectUIValidator directly?

Reply:
That is transitional and changes again in a later commit